source/js/buyers-guide/pni-sticky-nav.js

```diff
@@ -0,0 +1,21 @@
+const PNIStickyNav = {
+  init: () => {
+    const navContainer = document.querySelector(".primary-nav-container");
+
+    if (!navContainer) return;
+
+    let lastScroll = 0;
+    window.addEventListener("scroll", () => {
+      let currentScroll = window.pageYOffset;
+      if (currentScroll - lastScroll > 0 && currentScroll >= 300) {
```

Comment:
I'm also curious why the number `300` is chosen. Is it possible that we use JS to find out the value instead of hardcode it to 300?

Reply:
Honestly it was just a value I eyeballed and showed Rebecca since we don't want the nav to transform too much at the first bit of scrolling since it was a little distracting. Rebecca was good with the current setting. Should I just move it to a variable and comment it?